src/chain_spec.rs

```diff
@@ -136,7 +139,15 @@ fn general_testnet_genesis(
             code: general::WASM_BINARY.to_vec(),
             changes_trie_config: Default::default(),
         }),
-        asset: Some(GeneralConfig::AssetConfig {
+        asset: {
+
+            let mut file = File::open("~/src/data/currency_symbols.json").unwrap();
```

Comment:
This path won't work in production. Why not list all the codes in the chain spec JSON? This is why you have defined the `config` field in the first place.

Reply:
I have made the change now so this would no longer be an issue.